CHANGELOG.md

```diff
@@ -78,6 +78,8 @@
 ### 🐞 Fixed
 
 ### ⬆️ Improved
+- Updated the Compose framework version
+- Updated the Accompanist library version
```

Comment:
Let's add version numbers here, if they're relevant for our clients (or just omit these lines if not)